packages/image_picker/image_picker_platform_interface/lib/src/types/lost_data_response.dart

```diff
@@ -50,4 +56,9 @@ class LostDataResponse {
   final RetrieveType? type;
 
   bool _empty = false;
+
+  /// The list of files that were lost in a previous [getMultiImage] call due to MainActivity being destroyed.
+  ///
+  /// Can be null if [exception] exists.
```

Comment:
Should this be:
```suggestion
  /// Can be null if [exception] or [file] exists.
```

Similarly for the `file` property, but I can't add a comment there.

Reply:
I did the changes. But the unit test will be done on the Android PR.